downloads/README.md

```diff
@@ -21,8 +21,15 @@ of the ODP reference from Git**
 <th>
 </th>
 
-<th colspan="1" id="TH_17">odp-linux pktio types  
-(Socket, Netmap, DPDK)</th>
+<th colspan="1" id="TH_17">odp-linux
+pktio types:
+ - Socket
+ - Netmap
+ - DPDK
+ - TAP
+ - IPC
```

Comment:
Remove IPC. All others are well known technologies, but IPC is our own creation (it just raises questions, instead of giving answers here).